packages/react/src/css-idents.ts

```diff
@@ -341,4 +341,7 @@ export default {
   c7: "data-disabled",
   c8: "hover-enabled",
   c9: "aria-error",
+  ca: "data-disabled",
```

Comment:
let's drop the commit with these changes to `css-idents.ts` since they exist in `main` now